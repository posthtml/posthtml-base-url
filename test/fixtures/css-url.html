<head>
  <style>
    @font-face {
      font-family: 'Example';
      src: url('fonts/Example1.woff2') format('woff2'), url(fonts/Example2.woff) format('woff'), url("fonts/Example3.woff") format('woff');
    }

    @font-face {
      font-family: 'Example 2';
      src: url('https://preserve.me/fonts/Example1.woff2') format('woff2'), url(https://preserve.me/fonts/Example2.woff) format('woff');
    }

    .test {
      background-image: url('image.jpg');
      background: url('image.jpg');
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .test-2 {
      background-image: url("image.jpg");
      background: url("image.jpg");
      background-image: url("https://preserve.me/image.jpg");
      background: url("https://preserve.me/image.jpg");
    }

    .test-3 {
      background-image: url(image.jpg);
      background: url(image.jpg);
      background-image: url(https://preserve.me/image.jpg);
      background: url(https://preserve.me/image.jpg);
    }

    .multiple-bg {
      background-image: url('one.png'), url(two.png), url("three.png"), linear-gradient(to right, rgba(30, 75, 115, 1), rgba(255, 255, 255, 0));
    }
  </style>
</head>

<body style="background: url('image.jpg')">
  <style>
    @font-face {
      font-family: 'Example 2';
      src: url('fonts/Example3.woff2') format('woff2'), url('fonts/Example4.woff') format('woff');
    }
  </style>
  <table style="background: url(image.jpg)">
    <tr>
      <th style="background-image: url('image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(image.jpg)">Cell</td>
    </tr>
  </table>
</body>
