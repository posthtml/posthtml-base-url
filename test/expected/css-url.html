<head>
  <style>@font-face {
      font-family: 'Example';
      src: url('https://example.com/fonts/Example1.woff2') format('woff2'), url(https://example.com/fonts/Example2.woff) format('woff'), url("https://example.com/fonts/Example3.woff") format('woff');
    }

    @font-face {
      font-family: 'Example 2';
      src: url('https://preserve.me/fonts/Example1.woff2') format('woff2'), url(https://preserve.me/fonts/Example2.woff) format('woff');
    }

    .test {
      background-image: url('https://example.com/image.jpg');
      background: url('https://example.com/image.jpg');
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .test-2 {
      background-image: url("https://example.com/image.jpg");
      background: url("https://example.com/image.jpg");
      background-image: url("https://preserve.me/image.jpg");
      background: url("https://preserve.me/image.jpg");
    }

    .test-3 {
      background-image: url(https://example.com/image.jpg);
      background: url(https://example.com/image.jpg);
      background-image: url(https://preserve.me/image.jpg);
      background: url(https://preserve.me/image.jpg);
    }

    .multiple-bg {
      background-image: url('https://example.com/one.png'), url(https://example.com/two.png), url("https://example.com/three.png"), linear-gradient(to right, rgba(30, 75, 115, 1), rgba(255, 255, 255, 0));
    }</style>
</head>

<body style="background: url('https://example.com/image.jpg')">
  <style>@font-face {
      font-family: 'Example 2';
      src: url('https://example.com/fonts/Example3.woff2') format('woff2'), url('https://example.com/fonts/Example4.woff') format('woff');
    }</style>
  <table style="background: url(https://example.com/image.jpg)">
    <tr>
      <th style="background-image: url('https://example.com/image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(https://example.com/image.jpg)">Cell</td>
    </tr>
  </table>
</body>
